### DISCUSSION TITLE
[webapp] Issue starting latest containers due to using rootless port 80

### DISCUSSION THREAD

**Discussion by michaeljguarino** [airbytehq/airbyte#30284] [Connector Questions]
It looks like newer versions of airbyte webapp might have pulled an nginx version that doesn't run as root anymore and thus cannot start on port 80.  I don't think there's a workaround because the nginx is hardcoded here: https://github.com/airbytehq/airbyte-platform/blob/main/airbyte-webapp/nginx/default.conf.template#L14.  Anyone have an idea of how to fix?

**Comment by truonglac2603** [comment]
i'm facing the same issue

**Comment by marcosmarxm** [comment]
@michaeljguarino and @truonglac2603 where are you running the Airbyte platform? Can you give more info about the steps you
re trying to use to run Airbyte? Are you changing the .env or docker-compose file?

**Reply by truonglac2603** [reply]
I'm currently deploying Airbyte ver 0.50.43 on a Linux server, following instructions from docs to deploy on Local machine with Docker. All i'd done is `sudo apt install gnome-keyring` to solve a node-keytar problem. Then when i ran `./run-ab-platform.sh` again, every container ran well except webapp with this issue:  
  starting nginx...
  airbyte-proxy                     | nginx version: nginx/1.25.2
  airbyte-webapp                    | 2024/01/18 01:25:47 [emerg] 1#1: socket() [::]:80 failed (97: Address family not supported by protocol)
  airbyte-webapp                    | nginx: [emerg] socket() [::]:80 failed (97: Address family not supported by protocol)
Any idea how i could solve this? Thanks in advance!

**Reply by truonglac2603** [reply]
@marcosmarxm for more detail, this is the old discussion of this issue [https://discuss.airbyte.io/t/airbyte-webapp-not-starting/3348](url)

[a comment by an automated account is not shown]
